src/web-extension-api.js

```diff
@@ -215,6 +215,7 @@ export default {
     switch (message.type) {
       case "update-state": {
         // update the UI.
+        message.data.availableStudies = message.data.availableStudies.filter(study => !study.studyPaused);
```

Comment:
Since we're not changing the UI (other than hiding studies) based on the paused/resumed stated, let's not leak this logic in the UI. Let's do it in [the function to send the updates](https://github.com/mozilla-rally/rally-core-addon/blob/5faf1f355bf25c92189725725d018b0d7c5a69cb/core-addon/Core.js#L567)